oci/private/tarball.sh.tpl

```diff
@@ -3,24 +3,32 @@ set -o pipefail -o errexit -o nounset
 
 readonly STAGING_DIR=$(mktemp -d)
 readonly YQ="{{yq}}"
+readonly TAR="{{tar}}"
 readonly IMAGE_DIR="{{image_dir}}"
-readonly BLOBS_DIR="${STAGING_DIR}/blobs"
 readonly TARBALL_PATH="{{tarball_path}}"
 readonly TAGS_FILE="{{tags}}"
 
+# Write tar manifest in mtree format
+# https://man.freebsd.org/cgi/man.cgi?mtree(8)
+# so that tar produces a deterministic output.
+mtree=$(mktemp)
+function add_to_tar() {
+    tar_path=$1
+    content=$2
+    readonly mtree_flags="uid=0 gid=0 mode=0755 time=1672560000 type=file"
+    echo >>"${mtree}" "blobs/${LAYER}.tar.gz ${mtree_flags} content=${IMAGE_DIR}/blobs/${LAYER}"
```

Comment:
You could use $tar_path and $content, I guess.

Reply:
yup sorry I've been pushing before running tests locally,  don't normally have such an active volunteer reviewer 😅